server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileService.java

```diff
@@ -94,14 +89,6 @@ public void backup(String profileKey, Writer writer) {
     backuper.backup(profileKey, writer);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "backup"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)